### ISSUE TITLE
Kaputte Links im Widget

### ISSUE THREAD

**Issue by Zodiac1978** [deworg/wpmg-widget#20]
So langsam starten die Meetups wieder! Zeit für ein Aufräumen im Widget:

Nach einem Linkcheck sieht es wie folgt aus:

| Ort | Linkstatus |
| --- | --- |
| WP Meetup Aachen | Meetup.com OK |
| WP Meetup Berlin | Website OK |
| WP Meetup Bern | Website OK |
| WP Meetup Bonn | Website OK |
| WP Meetup Bremen | Website Zertifikatsproblem |
| WP Meetup Dresden | Website Domainproblem |
| WP Meetup Düsseldorf | Meetup.com OK |
| WP Meetup Eifel | Meetup.com OK |
| WP Meetup Franken | Website Weiterleitung zu https://wpmeetup-nuernberg.de/ |
| WP Meetup Frankfurt | Website OK |
| WP Meetup Fulda | Meetup.com 404 |
| WP Meetup Hannover | Meetup.com OK |
| WP Meetup Karlsruhe | Website OK |
| WP Meetup Koblenz | Website OK |
| WP Meetup Köln | Website OK |
| WP Meetup Leipzig | Website gelöscht (wp.com) |
| WP Meetup Mannheim | Website OK |
| WP Meetup München | Website OK |
| WP Meetup Neustadt | Meetup.com OK |
| WP Meetup Osnabrück/Münster | Website Domainproblem |
| WP Meetup Paderborn | Website Domainproblem |
| WP Meetup Potsdam | Website OK |
| WP Meetup Region 38 | Meetup.com 404 |
| WP Meetup Rostock | Website OK |
| WP Meetup Saarland | Website Domainproblem |
| WP Meetup Stuttgart | Website OK |
| WP Meetup Würzburg | Website Domainproblem |
| WP Meetup Wien | Website Weiterleitung zu https://wp-network.org |
| WP Meetup Zürich | Meetup.com OK |

**Comment by Zodiac1978** [comment]
Nach einem Abgleich https://wpmeetups.de/meetup/ scheinen (unabhängig von deren Status) noch folgende Meetups zu existieren (oder mal existiert zu haben/geplant gewesen sein):

WP Meetup Braunschweig-Wolfsburg
WP Meetup Dortmund
WP Meetup Freiburg
WP Meetup Halle
WP Meetup Heidelberg
WP Meetup Jena
WP Meetup Kassel
WP Meetup Kiel
WP Meetup Nürnberg
WP Meetup Regensburg

**Comment by christopherkurth** [comment]
Ja, WP Meetup Jena ist existent und ist unter https://wpjena.de erreichbar.

**Comment by Zodiac1978** [comment]
Das automatische Auslesen der Meetups von wpmeetups.de scheint kaputt zu sein.

Der Aufruf von https://wpmeetups.de/wp-json/wp/v2/meetup/?per_page=100&orderby=title&order=asc endet bei mir in einem kritischen Fehler.

**Comment by staude** [comment]
Würzburg redirected auf Meetup.com, aber nur http, https ist defekt.

**Comment by espiat** [comment]
hab nachgeschaut. die daten die dort ausgegeben werden, kommen aus einem transient.
wird der spätestens mit einem DB clean gelöscht, dann können keine neuen Daten mehr generiert werden, weil der JSON einen 
`HTTP/1.1 500 Internal Server Error
`
zurückgibt.

erst muss der json repariert werden.

**Comment by Zodiac1978** [comment]
Ja, das ist mir auch aufgefallen. Das muss Simon auf der Website reparieren. Er ist schon informiert!

**Comment by danpardella** [comment]
Leipzig kriegt eine neue Website. Bis dato gern meetup.com ;-)

**Comment by Zodiac1978** [comment]
Ich schlage vor, dass wir im Plugin prüfen, ob der Request erfolgreich war (Status 200) und falls nicht einen Fehler/Notice ausgeben:

https://github.com/deworg/wpmg-widget/blob/master/wpmeetup-widget.php#L57-L79

**Comment by 2ndkauboy** [comment]
Das sollten wir auf jeden Fall tun! Ich kann es mir mal anschauen.

**Comment by 2ndkauboy** [comment]
Zu WP Meetup Braunschweig-Wolfsburg habe ich nur diese FB Gruppe gefunden: https://www.facebook.com/groups/642749572507150/

**Comment by 2ndkauboy** [comment]
Wien würde ich auch erst einmal rausnehmen, da es kein WP Meetup (der Foundation) mehr ist :thinking:

**Comment by Zodiac1978** [comment]
@2ndkauboy Die Anbindung an die API ist ja kaputt gegangen, daher ist das alles nicht der neueste Stand. Warten wir ab, was passiert wenn die API wieder läuft und wir die Daten von wpmeetups.de haben. Vorher würde ich nicht einzelne Links recherchieren.

**Comment by 2ndkauboy** [comment]
Habe ich in #27 schon gemacht ;)
